### DISCUSSION TITLE
Is there any start guide to work with EKS NLB?

### DISCUSSION THREAD

**Discussion by davidt-gh** [nginx/kubernetes-ingress#6284] [Q&A]
I just want to start a new NLB that using my NGINX ingress controller, and getting health check fail issue.
I don't really came to discussion to figure out how to resolve my problem, but better to understand what steps need to do from scratch.
The baseline I understand is:
1. New EKS
2. VPC CNI addon
3. Install  aws-load-balancer-controller
4. Install nginx ingress controller and annoation it to be NLB

Let me know if one of this steps is wrong and where can I see basic configuration to start.

**Comment by AlexFenlon** [comment]
Hi @davidt-gh,

What version of NIC are you using?

What type of health check are you using?

Can you send us your logs to show the error.
